index.js

```diff
@@ -67,7 +93,14 @@ function getAverageIMDBRating() {}
  *  countByRating(movies);
  *  //> { G: 3, PG: 7 }
  */
-function countByRating() {}
+function countByRating(movies) {
+  let movType = {};
+  for(let i = 0; i < movies.length; i++){
+    movType[movies[i].rated] = (movType[movies[i].rated] || 0) + 1
```

Comment:
Very slick short circuit - just make sure we know what the left side of it would return if we DO NOT have movType[movies[i].rated]